buildIncremental.fsx

```diff
@@ -9,6 +9,8 @@ open Fake.Git
 
 module IncrementalTests =   
 
+    let akkaDefaultBranch = "v1.3"
```

Comment:
yep, this looks fine. Won't need to change again after we merge `v1.3` into dev.